src/schema/v2/partner/__tests__/partnerMerchantAccountsConnection.test.ts

```diff
@@ -0,0 +1,84 @@
+import { runQuery } from "schema/v2/test/utils"
+import gql from "lib/gql"
+
+describe("partner.merchantAccountsConnection", () => {
+  let response
+  let context
+
+  beforeEach(() => {
+    response = [
+      {
+        external_id: "stripe-account-1",
+      },
+    ]
+    const partnerMerchantAccountsLoader = () => {
+      return Promise.resolve({
+        body: response,
+        headers: {
+          "x-total-count": response.length,
+        },
+      })
+    }
+
+    context = {
+      partnerLoader: () => {
+        return Promise.resolve({
+          _id: "partnerID",
+        })
+      },
+      partnerMerchantAccountsLoader: partnerMerchantAccountsLoader,
+    }
+  })
+
+  it("returns merchant accounts", async () => {
+    const query = gql`
+      {
+        partner(id: "partnerID") {
+          merchantAccountsConnection(first: 1) {
+            edges {
+              node {
+                externalId
+              }
+            }
+          }
+        }
```

Comment:
**Question:** are we going to get multiple accounts? since in the PR description it says we'll always return the first one

In case we'll always return one account, my suggestion, for simplicity, would be to return the external id instead of an array